composer.json

```diff
@@ -29,10 +29,10 @@
       }
     },
     "require-dev": {
-        "phpunit/phpunit": "^5"
+        "phpunit/phpunit": "^8.0"
     },
     "require": {
-        "imjoehaines/flowder": "^1.0",
-        "codeception/codeception": "^2.1"
+        "imjoehaines/flowder": "^2.0",
```

Comment:
This only allows PHP 7.3 or 7.4; do you need it to run on PHP 8 too?

Reply:
7.4 should be enough for us, for now